tests/stress_tests/common/ie_pipelines/pipelines.cpp

```diff
@@ -97,14 +95,14 @@ infer_request_inference(const std::string &model, const std::string &target_devi
 
 
 std::function<void()> reinfer_request_inference(std::shared_ptr<InferApiBase> &ie_wrapper) {
-    return [&] {
+    return [=] {
         ie_wrapper->infer();
```

Comment:
I am not sure if it's right here, from my understand of the test code, maybe I am wrong, even if use [=] here, when test_params.numthreads > 1, there will be multi threads use same ie_wrapper to call infer(); one of them will do infer and other threads will throw request busy. so please help to set test_params.numthreads=10 and check if the test case is ok, thanks.

Reply:
Thank you for detailed review! The original issue was in consecutive inference, each inference request in separate thread (code snippet may be found in CVS-111490). So it shouldn't cause issues with `ie_wrapper`

`test_params.numthreads` is a legacy from `StressUnitTests`, `StressMemLeakTests` weren't designed to test parallel execution because it is non-trivial